src/aiff.c

```diff
@@ -1059,6 +1059,13 @@ aiff_read_comm_chunk (SF_PRIVATE *psf, COMM_CHUNK *comm_fmt)
 	else
 		psf_log_printf (psf, "  Sample Size : %d\n", comm_fmt->sampleSize) ;
 
+
+	if (psf->sf.channels != comm_fmt->numChannels && psf->peak_info)
```

Comment:
```suggestion
	if ((psf->sf.channels != comm_fmt->numChannels &&) psf->peak_info)
```
I like not having to think about precedence of binary operations

Reply:
What about 
```c
	if ((psf->sf.channels != comm_fmt->numChannels) && psf->peak_info)
```
?

Reply:
see, proving my case :D

Reply:
yes, mine obviously can't work :wink: